packages/geo/src/lib/filter/ogc-filterable-item/ogc-filterable-item.component.ts

```diff
@@ -79,6 +80,15 @@ export class OgcFilterableItemComponent implements OnInit {
   ) {}
 
   ngOnInit() {
+    if (
```

Comment:
Create a const ogcFilters = this.dataSource.options.ogcFilters. This will shorten thse lines a lot.